src/webextension/list/components/item-fields.css

```diff
@@ -17,6 +17,18 @@
   resize: none;
 }
 
+.item-fields input {
+  max-width: 242px;
```

Comment:
Could we change this to `.item-fields .input` (or even better, simply `.input` like above)? Then you'd just need to pass `className` to all the inputs in `item-fields.js`.

I know this is a really minor thing, but technically, the `input` element is contained within a different React component, and it's not actually a part of that component's public API (in other words, we might change the implementation of that component one day in a way that breaks this). However, passing `className` to override some of the default styles for that component *is* part of the public API, and we make a guarantee that that should always work.